oak_utils/src/lib.rs

```diff
@@ -204,3 +204,37 @@ where
         .compile_protos(inputs, includes)
         .expect("could not run prost-build");
 }
+
+/// Generate Protobuf code using `tonic`.
+/// `build_client` and `build_server` specify whether to generate client or server related code.
+fn compile_proto(
+    proto_path: &str,
+    file_path: &str,
+    build_client: bool,
+    build_server: bool,
```

Comment:
Perhaps define two enums `BuildClient { Yes, No }` and `BuildServer { Yes, No }` (or similar) instead of vague bools? In fact then you can just expose this function, you don't need to have the helper function to make it actually readable.

Reply:
[See also](https://github.com/project-oak/oak/pull/1108#discussion_r445413379)

Reply:
I think David's comment applies here. Maybe we can use this PR to introduce a configuration object for compiling protos (optionally with tonic)? Also fine to defer that to a later PR.

I would recommend to go with a builder pattern if you do decide to go with a config struct, that way you can easily add new fields later without changing example code, which I think happens quite frequently now.